### PR TITLE
invalid configuration file bug corrected and custom separator entry

### PR DESCRIPTION
**Pull request by ftapajos**
- Adds a valid configuration file check
- Creates a custom "separator" entry and its test file

### CODE REVIEW

File: qtws.cpp [hunk 1]
```diff
@@ -8,6 +8,8 @@
 #include <QRegExp>
 #include <QStringList>
 
+#include <iostream>
```
Comment (intersimone999) [inline qtws.cpp]: Is this actually necessary? Can we remove it?

**Review by intersimone999 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution. Can you please check my comment about the "include" line? Besides that, it seems fine to me.

**Comment by ftapajos** [comment]
Please do remove it, I used it for a test and I forgot to remove it

-------- Mensagem original --------
De : Simone Scalabrino <notifications@github.com>
Data: 26/01/2019 08:44 (GMT-03:00)
Para: intersimone999/qtws <qtws@noreply.github.com>
Cc: ftapajos <ftapajos@gmail.com>, Author <author@noreply.github.com>
Assunto: Re: [intersimone999/qtws] invalid configuration file bug corrected and custom separator entry (#12)


@intersimone999 requested changes on this pull request.

Thanks a lot for your contribution. Can you please check my comment about the "include " line? Besides that, it seems fine to me.

________________________________

In qtws.cpp<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fintersimone999%2Fqtws%2Fpull%2F12%23discussion_r251195510&data=02%7C01%7C%7Cd65bf110c2584f9dc01008d6837b4270%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636840962743778063&sdata=YYrfJKVfn8HHwkG2fzxsyeOxNtZnQcP6gblWOBHeEqk%3D&reserved=0>:

> @@ -8,6 +8,8 @@
 #include <QRegExp>
 #include <QStringList>

+#include <iostream>


Is this actually necessary? Can we remove it?

File: qtws.cpp (hunk 1 above)
Comment (ftapajos) [reply qtws.cpp]: Sure. My mistake.

**Comment by intersimone999** [comment]
Thanks.

**Pull request merged**